src/auro-radio-group.js

```diff
@@ -100,6 +120,33 @@ export class AuroRadioGroup extends LitElement {
     this.addEventListener('resetRadio', this.resetRadio);
     this.addEventListener('auroRadio-blur', this.handleRadioBlur);
     this.addEventListener('auroRadio-selected', this.handleSelection);
+    this.addEventListener('auroFormElement-validated', this.handleValidation);
+  }
+
+  firstUpdated() {
+    // Add the tag name as an attribute if it is different than the component name
+    this.runtimeUtils.handleComponentTagRename(this, 'auro-radio-group');
+  }
+
+  /**
+   * Method for handling of validation of the radio group.
+   * @private
+   * @param {Event} event - The trigger event tied to this function.
+   * @returns {void}
+   */
+  handleValidation(event) {
+    this.validity = event.detail.validity;
```

Comment:
This should already be being applied by the validation library and is redundant here.